packages/react-vapor/src/components/navigation/pagination/NavigationPaginationReducers.ts

```diff
@@ -37,6 +39,10 @@ export const paginationReducer = (
                 : state;
         case FilterActions.filterThrough:
             return s.contains(state.id, action.payload.id) ? {...state, pageNb: 0} : state;
+        case ListBoxActions.select: {
+            const tableId = TableHOCUtils.getTableIdFromPredicateId(action.payload.id);
+            return tableId && s.contains(state.id, tableId) ? {...state, pageNb: 0} : state;

```

Comment:
Since the real method in underscore string library is `.include` from which `.contains` is only an aliase. I would use the real method name IMO. it's only a detail though.

Reply:
ah you have a good point! but in this file I think I would keep .contains, because I've followed the filterThrough. I'm not sure if I must change all to .include or it's simply kind of convention to use .contains hmm